packages/plugins/RedPacket/src/SiteAdaptor/FireflyRedPacketAccountItem.tsx

```diff
@@ -0,0 +1,55 @@
+import { memo, useCallback } from 'react'
+import { Box, Typography } from '@mui/material'
+import { formatEthereumAddress, isValidAddress } from '@masknet/web3-shared-evm'
+import { makeStyles } from '@masknet/theme'
+import { Icons } from '@masknet/icons'
+import { ENS , EVMExplorerResolver } from '@masknet/web3-providers'
+import { openWindow } from '@masknet/shared-base-ui'
+
+const useStyles = makeStyles()((theme) => ({
+    linkIcon: {
+        color: theme.palette.maskColor.second,
+        height: 16,
+        width: 16,
+    },
+    linkButton: {
+        cursor: 'pointer',
+        background: 'none',
+        border: 'none',
+        padding: 0,
+        height: 16,
+    },
+}))
+
+interface Props {
+    addressOrEns: string
```

Comment:
Does it could be other kind of domain name?

Reply:
Creator only has two types address or ens